### PR TITLE
Add avatars to authors page

### PR DESCRIPTION
**Pull request by BasThomas**
Ref #175 
Using the GitHub avatars right now.

screen shot 2017-01-30 at 21 52 26

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -187,3 +187,7 @@ blockquote {
 .post-count {
    display: inline-block;
 }
+
+.img-circle {
```
Comment (jessesquires) [inline css/style.css]: this can be removed. `img-circle` is defined in bootstrap:

http://getbootstrap.com/css/#images

**Comment by jessesquires** [comment]
Thanks @BasThomas ! Looks awesome.

Let's see if we can align the images to the *left* of the text.

Bootstrap's "Media Objects" will probably work:
http://getbootstrap.com/components/#media

**Comment by jessesquires** [comment]
✨ friendly ping ✨ 

Hey @BasThomas 😄 still want to work on this? Just checking in, no rush!

**Comment by BasThomas** [comment]
Yeah, let me take a stab at this soon. :)

**Comment by BasThomas** [comment]
screen shot 2017-03-17 at 09 45 59

**Comment by jessesquires** [comment]
@BasThomas nice! 🎉 

I think it's definitely better than avatars on top. A few improvements:

- more vertical spacing between each author?
- can we left-align the issue lists with the author name?

**Comment by jessesquires** [comment]
ahhh fuck. i just fucked up this branch trying to rebase

**Comment by jessesquires** [comment]
@BasThomas - if you still have your branch on your machine, we can close this PR, remove the branch on GH, then you can re-push and open a new PR.

sorry about that 😞 😢

**Pull request closed without merging**